lib-ui-core/src/main/java/com/pax/us/pay/ui/core/helper/SearCardHelper.java

```diff
@@ -45,6 +45,7 @@ public void setSecurityArea(int x, int y, int width, int height, int fontSize) {
         bundle.putInt(EntryRequest.PARAM_WIDTH, width);
         bundle.putInt(EntryRequest.PARAM_HEIGHT, height);
         bundle.putInt(EntryRequest.PARAM_FONT_SIZE, fontSize);
+        bundle.putString(EntryRequest.PARAM_COLOR, "FF0000");
         super.setSecurityArea(bundle);
```

Comment:
Should not hard-code color here.

Reply:
hardcode issue has been fixed. https://github.com/PAXTechnologyInc/POSLink-UI/tree/develop